### PR TITLE
Update to fix subdirectory install path problem.

### PR DESCRIPTION
**Pull request by Hazlitte**
## Summary

Update to fix subdirectory install path problem.

Addresses issue #2821 

## Checklist

- [X] My code is tested and passes existing unit tests.
- [X] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [X] I have added a QA Brief on the issue linked above.
- [X] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/components/data-table.js [hunk 1]
```diff
@@ -50,8 +51,13 @@ export const getDataTableFromData = ( data, headers, options ) => {
...
 		const link = links && links[ j ];
-		const permaLink = link && link[ 0 ] === '/' ? global._googlesitekitLegacyData.admin.siteURL + link : link;
-
+		// const oldLink = link && link[ 0 ] === '/' ? global._googlesitekitLegacyData.admin.siteURL + link : link;
+		// console.log( 'SiteURL', global._googlesitekitLegacyData.admin.siteURL );
+		// console.log( 'path', link );
+		// console.log( 'oldlink', oldLink );
+		const permaLink = link && link[ 0 ] === '/' ? getFullURL( global._googlesitekitLegacyData.admin.siteURL, link ) : link;
+		// console.log( 'newLink', permaLink );
+		// console.log( 'URL', getFullURL( 'https://www.example.com/', '' ) );
```
Comment (tofumatt) [inline assets/js/components/data-table.js:60]: 🔥 

```suggestion
		const permaLink = link && link[ 0 ] === '/' ? getFullURL( global._googlesitekitLegacyData.admin.siteURL, link ) : link;
```

File: assets/js/util/getFullURL.test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		).toEqual( 'https://www.example.com/' );
+
+		expect(
+			getFullURL( 'https://www.example.com', '/path' )
+		).toEqual( 'https://www.example.com/path' );
+
+		expect(
+			getFullURL( 'https://www.example.com/slug/slug', '/path' )
+		).toEqual( 'https://www.example.com/path' );
+
```
Comment (tofumatt) [inline assets/js/util/getFullURL.test.js:37]: I added this test locally and it passed, which makes sense, so I added it explicitly so a URL with a port number in it is handled:

```js
expect(getFullURL( 'https://www.example.com:444/slug/slug', '/path' ) )
  .toEqual( 'https://www.example.com:444/path' );
```

**Review by tofumatt — COMMENTED** [review]
Looks good; I added a few extra tests to clarify URLs with port numbers and see there's some console log statements to remove, but I'll remove them then get this merged 👍🏻

**Comment by tofumatt** [comment]
@googlebot I consent.

**Review by tofumatt — APPROVED** [review]

File: assets/js/util/getFullURL.test.js (hunk 2 above)
Comment (Hazlitte) [reply assets/js/util/getFullURL.test.js:37]: I left the console logs in there 🤦‍♂️.  Trying to do too many things at once. Doh!
Adding a port test was a really good idea.  I didn't think of that.

**Pull request merged**